kubernetes/policies/pss/Baseline #9 - Unsafe sysctl options set.rego

```diff
@@ -7,12 +7,12 @@ default failSysctls = false
 
 __rego_metadata__ := {
 	"id": "KSV026",
-	"title": "Unsafe sysctl options set",
+	"title": "Sysctls configures an 'unsafe' value",
 	"version": "v1.0.0",
 	"severity": "MEDIUM",
 	"type": "Kubernetes Security Check",
-	"description": "Sysctls can disable security mechanisms or affect all containers on a host, and should be disallowed except for an allowed 'safe' subset. A sysctl is considered safe if it is namespaced in the container or the pod, and is isolated from other pods and processes on the same node.",
-	"recommended_actions": "Do not set 'spec.securityContext.sysctls' or set to values in allowed subset.",
+	"description": "Sysctls can disable security mechanisms or affect all containers on a host, and should be disallowed except for an allowed 'safe' subset. A sysctl is considered safe if it is namespaced in the container or the Pod, and it is isolated from other Pods or processes on the same Node.",
+	"recommended_actions": sprintf("Do not set 'spec.securityContext.sysctls' or set to values in an allowed subset, %s", [allowed_sysctls]),
```

Comment:
@knqyf263 will using sprintf and in policy evaluation be a common theme going forwards for rego metadata? So far I've assumed that these are just plain strings and are not evaluated in anyway. For tools like AVD the simplicity of being a string makes using plain regex easier. But if the policy needs to be evaluated prior to generating the full metadata then maybe we need to use a proper Rego engine and load the policy somehow to get the description. 

Personally I would vote for the simpler option which would be a simple string like:
```
"Do not set 'spec.securityContext.sysctls' or set to values in an allowed subset: X policy, Y policy, Z policy"
```

What do you think?